src/views/Lottery/components/ClaimPrizesModal/ClaimPrizesInner.tsx

```diff
@@ -138,7 +139,9 @@ const ClaimInnerContainer: React.FC<ClaimInnerProps> = ({ onSuccess, roundsToCla
         }
       } catch (error) {
         console.error(error)
+        setPendingTx(false)
         toastError(t('Error'), t('%error% - Please try again.', { error: error.message }))
+        break
```

Comment:
Why the break?

Reply:
It's just to break out of the loop that triggers one claim transaction after another - without it, if you click claim > then cancel transaction > loop still moves onto next transaction in batch.